apps/wallet-mobile/src/features/Swap/common/SwapFormProvider.tsx

```diff
@@ -325,12 +325,14 @@ const swapFormReducer = (state: SwapFormState, action: SwapFormAction) => {
     switch (action.type) {
       case SwapFormActionType.SellTouched:
         draft.sellQuantity.isTouched = true
+        draft.sellQuantity.displayValue = ''
```

Comment:
Did product request it? That's sad. We were recovering from any inputted number.

Reply:
This happens before the calculations, resets the field, but then it's calculated by the swap state and we get the value. It just removes an edge case the was causing some confusion.